add optional param

cn-tools opened the pull request:
- add argument '-x' to generate hashtags instead of normal tags
- add argument '-a' to use title of the first annotations, If no title is available in basic data

cn-tools committed:
add optional param to generate hashtags instad of semicolon-separated tags

cn-tools committed:
add argument 'a'

If no title is available in basic data, use title of the first annotations

erikelisath commented on keep-to-markdown.py:
```diff
@@ -220,6 +226,7 @@ def create_folder():
     parser.add_argument('-i', metavar='PATH', required=True, help='The path to the Takeout folder.')
     parser.add_argument('-t', action='store_true', help='Use folders instead of front-matter for tags.')
     parser.add_argument('-x', action='store_true', help='Build hashtags from tags.')
+    parser.add_argument('-a', action='store_true', help='If no title is available in basic data, use title of the first annotations.')
```
this new argument is missing in the README file

erikelisath replied:
Did it on my own.

erikelisath committed:
Update README.md

add argument '-a' to use title of the first annotations, If no title is available in basic data

erikelisath commented:
Thanks mate!
